src/modules/roc_fec/idecoder.h

```diff
@@ -23,14 +23,31 @@ class IDecoder {
 public:
     virtual ~IDecoder();
 
+    //! Get the maximum value of a source block for a used scheme.
+    //!
+    //! @returns
+    //!  0 when there are no limits.
+    virtual size_t max_source_block() const = 0;
```

Comment:
Discussed to replace it with max_block_length().

Reply:
Done